decode.js

```diff
@@ -5,34 +5,51 @@ const mri = require('mri')
 const pkg = require('./package.json')
 
 const argv = mri(process.argv.slice(2), {
+    alias: {
```

Comment:
To be honest, I think this change decreases the readability of the code. I don't have a strong opinion though.

Reply:
Same, no strong opinion